libr/bin/p/bin_any.c

```diff
@@ -1,5 +1,6 @@
 /* radare - LGPL - Copyright 2009-2017 - pancake, nibble */
 
+#include <r_userconf.h>
```

Comment:
Already presents in r_types.h: https://github.com/radare/radare2/blob/master/libr/include/r_types.h#L6